doc/sslang-lrm.tex

```diff
@@ -525,6 +525,7 @@ \section{Expressions}
 \alt "if" <expr_blk> "{" <expr> "}" <expr_else>$^{?}$
 \alt "while" <expr_blk> "{" <expr> "}"
 \alt "fun" <pat>$^{+}$ "{" expr "}"
+\alt "match" <exprBlk> "{"< matchArms> "}"
```

Comment:
1. use consistent casing: `expr_blk`, `match_arms`.
2. you're missing the grammar for `match_arms`
3. honestly dont make a separate grammar for `match_arms`, look what I did for `wait` and `par`. Use regex syntax.

Reply:
I have changed the casing. I did include the grammar for match_arms: <match_arms> ::= match_arm "|" match_arms
\alt match_arm